Readme.md

```diff
@@ -371,6 +371,31 @@ An instance of `Client` is passed to the `soap.createClient` callback.  It is us
       // result is a javascript object
   })
 ```
+#### args
+
+The `args` argument allows you to supply arguments that generate an XML document inside of the SOAP Body section.
+
+##### Example with JSON as an argument
+may generate an XML in the SOAP Body such as:
+``` javascript
+  <nsPrefixInXsd:MyMessage xmlns:nsPrefixInXsd="http://www.example.com/v1/ns1">
+      <ChildElementNoNS>
+          <nsPrefixNOTInXsd:ChildElementNS nsPrefixNOTInXsd="http://www.example.com/v1/ns2">elementvalue</nsPrefixNOTInXsd:ChildElementNS>
+          <nsPrefixInXsd:ChildElementNS>elementvalue</nsPrefixInXsd:ChildElementNS>
+      </ChildElementNoNS>
+  </nsPrefixInXsd:MyMessage>
+```
+The namespaces and namespace prefixes should already be pre-populated from the WSDL/XSDs in the MyMessage and/or SOAP Envelope elements.  If an element in args contains no namespace prefix, the target namespace of MyMessage is used.  A prefix may or may not be added, depending on how the XSD is setup.
+Currently, elements may not contain both child elements and a text value.
+
+##### Example with XML String as an argument
+You may pass in a fully-formed XML string instead the individual elements and attributes that make up the XML.  The XML string should not contain an XML declaration (e.g., `<?xml version="1.0" encoding="UTF-8"?>`) or a document type declaration (e.g., `<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN" "http://www.w3.org/TR/html4/frameset.dtd">`).
+``` javascript
+  {_xml: "<nsPrefixInXsd:MyMessage xmlns:nsPrefixInXsd="http://www.example.com/v1/ns1">
```

Comment:
Please use simple qoutes(`'`) for defining the `xmlns:nsPrefixInXsd='http://www.example.com/v1/ns1'` or escape the double quotes `"` as otherwise the markup is broken (if you look at this file in the github diff-view, you see that the closing culry brace and everything after is marked "red", which basically means "bad markup")